MLOps.NET.Azure/Storage/StorageAccountModelRepository.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Azure.Storage.Blobs;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.Storage
+{
+    internal sealed class StorageAccountModelRepository : IModelRepository
+    {
+        private readonly BlobContainerClient blobContainerClient;
+        private const string _containerName = "model-repository";
```

Comment:
My bad, I should have added a coding style guide to the repo. Just to keep it consistent I think we will not be using underscore for private member variables (see blobContainerClient above).

Reply:
Done 👍